Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:current-alpine AS base
+WORKDIR /base
+COPY package*.json ./
+RUN npm install
```

Comment:
Very interesting, I decide to use` npm` as a default so that our `dockerfile` is surely worked. Anyway, I've to modify this  a bit to support `yarn`.